Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 ruby '2.1.2'
 
 # the most important stuff
-gem 'rails', '4.1.8'
+gem 'rails', '4.1.11'
```

Comment:
@ilnurnasyrov Upgrading to 4.2 is more complicated: http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html . In this PR I want only fix security vulnerabilities.